js/tests/unit/util/index.spec.js

```diff
@@ -213,6 +214,34 @@ describe('Util', () => {
         Util.typeCheckConfig(namePlugin, config, defaultType)
       }).toThrow(new Error('COLLAPSE: Option "parent" provided type "number" but expected type "(string|element)".'))
     })
+
+    it('should return null to string when null passed', () => {
```

Comment:
Can we reword this and the other description a little better please? :)

Reply:
reworded, do not hesitate if you have any suggestion of rewording